### PR TITLE
test for start jitsi button on event and project page

### PR DESCRIPTION
**Pull request by lokeshh**
Adds test for https://github.com/AgileVentures/WebsiteOne/issues/1726

### CODE REVIEW

File: features/step_definitions/jitsi_steps.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Then(/^I should see a start jitsi meet button$/) do
+  expect(page).to have_css '#start-jitsi'
+  # src = page.find(:css, '.btn-jitsi a')['src']
```
Comment (lokeshh) [inline features/step_definitions/jitsi_steps.rb]: @tansaku @mattlindsey Couldn't understand why the test is failing?

Comment (lokeshh) [inline features/step_definitions/jitsi_steps.rb]: Ah! Got it! It should be "Repeat Scrum" not "Scrum" because "Scrum" will run only for once.

**Comment by lokeshh** [comment]
@tansaku Ready to be merged

**Comment by tansaku** [comment]
thanks @lokeshh - great work here - I trust you've checked the tests fail when the functionality is removed :-)

also, can you update the description so that it says fixes for the correct issue?  Also, it's good practice to preface the branch with the issue number - check out: 

https://github.com/AgileVentures/WebsiteOne/blob/develop/CONTRIBUTING.md#pull-requests-naming-syncing-size

apologies if that feels like drudge work but it's a good pattern to get into: 

1) create branch prefixed with id of issue (moves issue into 'in progress')
2) create failing test on the branch (acceptance level)
3) create failing tests (unit level)
4) get test to pass with functionality
5) submit pull request with fixes #xyz 
6) pull request reviewed
7) changes made
8) pull request merged (presence of "fixes #xyz" then moves issue to done )
9) code moved to staging and checked against data clone
10) code moved to production

the use of the branch naming and the presence of the "fixes #xyz" all helps us try to keep the waffle board/github-issues up to date ...

Please do ask if you've got any questions about the flow, and see if you can bring it to mind on future work :-)

**Comment by lokeshh** [comment]
@tansaku This PR writes a test for #1726, although #1726 was fixed by #1731 but unfortunately not with tests (my mistake). What should I do in this case? I have added in the description `Adds test for #1726`. Is that enough? Or do I have a create a separate branch prefixed with 1726 and resubmit the PR?

**Pull request merged**